firefox-ios/firefox-ios-tests/Tests/XCUITests/FakespotTests.swift

```diff
@@ -95,18 +95,20 @@ class FakespotTests: BaseTestCase {
         waitUntilPageLoad()
         // The price tag icon is not displayed
         mozWaitForElementToNotExist(app.buttons[AccessibilityIdentifiers.Toolbar.shoppingButton])
-        // Open a product detail page and check the address bar
-        let searchField = app.webViews["contentView"].webViews.textFields["Search for anything"]
-        mozWaitForElementToExist(searchField)
-        searchField.tap()
-        searchField.typeText("Shoe")
-        mozWaitForElementToExist(app.webViews["contentView"].webViews.buttons["Search"])
-        app.webViews["contentView"].webViews.buttons["Search"].tap()
-        waitUntilPageLoad()
-        app.webViews["contentView"].links.element(boundBy: 7).tap()
-        waitUntilPageLoad()
-        // The price tag icon is not displayed
-        mozWaitForElementToNotExist(app.buttons[AccessibilityIdentifiers.Toolbar.shoppingButton])
+        if #available(iOS 17, *) {
```

Comment:
The `searchField` can't be typed on iOS 16 only.

Reply:
then you may need a if not available iOS 16, otherwise is not going to work for iOS 15

Reply:
iOS 15 has the same issue as iOS 16 does. This section of the code does not work on iOS 15.

(I will have another PR up for iOS 15.)